src/po/Make_mvc.mak

```diff
@@ -47,9 +43,9 @@ ICONV="$(GETTEXT_PATH)\iconv.exe"
 # If the "touch" program is installed on the system, but it is not registered
 # in the %PATH% environment variable, then specify the full path to this file.
 !IF EXIST ("touch.exe")
-TOUCH = touch.exe $@
+TOUCH_TARGET = touch.exe $@
 !ELSE
-TOUCH = @if exist $@ ( copy /b $@+,, ) else ( type nul >$@ )
+TOUCH_TARGET = @if exist $@ ( copy /b $@+,, ) else ( type nul >$@ )
 !ENDIF
```

Comment:
To make it work for both target files and dependencies let's change it this way:
```
!IF EXIST ("D:\Programs\Git\usr\bin\touch.exe")
TOUCH = "D:\Programs\Git\usr\bin\touch.exe" %1
!ELSE
TOUCH = if exist %1 ( copy /b %1+,, ) else ( type nul >%1 )
!ENDIF
```
To make it work for both target files and dependencies let's change it this way:

Then the call to it needs to be changed as well. In this case, it will be called using the "inline file" mechanism.
That is
```
target : dependence
       <<touch.bat [$@ | $?]
         $TOUCH
<<
```